packages/react/README.md

```diff
@@ -54,6 +54,57 @@ function SignInButton({ wallet }) {
 }
 ```
 
+### `useSignAndSendTransactions(uiWalletAccount, chain)`
+
+Given a `UiWalletAccount` and a chain that begins with `solana:`, this hook returns a function you can call to sign and send one or more serialized transactions in a single request.
+
+#### Arguments
+
+One or more config objects with the following properties:
+
+- `options`: An object with the following properties:
+    - `minContextSlot`: A slot at which any blockhash/nonce in the transaction is known to exist. This may be used by the signer and/or RPC to determine the validity of the blockhashes/nonces it has observed.
+- `transaction`: A `Uint8Array` of bytes that conforms to the [Solana transaction schema](https://solana.com/docs/core/transactions#transaction)
+
+#### Returns
+
+An array of objects with the following properties:
+
+- `signature`: A `Uint8Array` of bytes representing the signature of each sent transaction.
+
+#### Example
+
+```tsx
+import { getBase58Decoder } from '@solana/codecs-strings';
+import { useSignAndSendTransactions } from '@solana/react';
+
+function SignAndSendTransactionsButton({ account, transactionBytes1, transactionBytes2 }) {
+    const signAndSendTransactions = useSignAndSendTransactions(account, 'solana:devnet');
+    return (
+        <button
+            onClick={async () => {
+                try {
+                    const [first, second] = await signAndSendTransactions(
+                        { transaction: transactionBytes1 },
+                        { transaction: transactionBytes2 },
+                    );
+                    const [firstSignature, secondSignature] = [first.signature, second.signature].map(signature =>
+                        getBase58Decoder().decode(signature),
+                    );
+                    window.alert(
+                        `View transactions: https://explorer.solana.com/tx/${firstSignature}?cluster=devnet and https://explorer.solana.com/tx/${secondSignature}?cluster=devnet`,
+                    );
+                } catch (e) {
+                    console.error('Failed to send transactions', e);
```

Comment:
Note for future readers: this is a lie. The underlying API does not let you know which transaction failed to send; some of them might, in fact, have landed.